packages/flutter/lib/src/material/color_scheme.dart

```diff
@@ -19,22 +19,45 @@ class ColorScheme with Diagnosticable {
   /// Create a ColorScheme instance.
   const ColorScheme({
     required this.primary,
-    required this.primaryVariant,
-    required this.secondary,
-    required this.secondaryVariant,
-    required this.surface,
-    required this.background,
-    required this.error,
     required this.onPrimary,
+    Color? primaryContainer,
+    Color? onPrimaryContainer,
+    required this.secondary,
     required this.onSecondary,
-    required this.onSurface,
-    required this.onBackground,
+    Color? secondaryContainer,
+    Color? onSecondaryContainer,
+    Color? tertiary,
+    Color? onTertiary,
+    Color? tertiaryContainer,
+    Color? onTertiaryContainer,
+    required this.error,
     required this.onError,
+    Color? errorContainer,
+    Color? onErrorContainer,
+    Color? outline,
+    required this.background,
+    required this.onBackground,
+    required this.surface,
+    required this.onSurface,
+    Color? surfaceVariant,
+    Color? onSurfaceVariant,
+    Color? inverseSurface,
+    Color? inverseOnSurface,
+    Color? inversePrimary,
+    Color? shadow,
```

Comment:
nit: reorder the last 5 to match spec
<img width="145" alt="Screen Shot 2021-11-11 at 13 02 11" src="https://user-images.githubusercontent.com/6655696/141294826-ccf78ba2-de2f-4492-b438-63f47a2dc7f6.png">